### DISCUSSION TITLE
Version control with created models

### DISCUSSION THREAD

**Discussion by FosterChad95** [wpengine/atlas-content-modeler#623] [Q&A / Help / Support] [answered]
Hello! I am new to atlas content modeler and am using it in a project with version control. I am wondering if there is anyway to add version control when adding/deleting/editing a model? Kind of like how ACF fields have the acf-json. So far I have not found a way. Thank you

**Comment by mindctrl** [comment, marked as answer]
@FosterChad95 Thanks for reaching out. At this time ACM does not support a json version control feature like ACF does. You could export the models under Content Modeler > Tools, but you'd still need to deal with the version control yourself. The idea is on our backlog, but I don't know if/when we'll get to that feature.
